lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -49,15 +36,6 @@ $panel-properties-height: 56px !default;
 
         #{ms.$mat-expansion-panel-body} {
```

Comment:
maybe it's possible to get rid of the panel padding as well somehow? changing the container component somehow, etc?